frontend/src/entities/user/user.reducer.ts

```diff
@@ -28,9 +28,9 @@ const initialState = {
 };
 
 // Actions
-export const getPageSizeForUser = (user : IUser, tableName : string) => {
-    const pageSizes : Map<string, number> = new Map(Object.entries(user.preferences.pageSize || {}));
-    if (pageSizes?.has(tableName)) {
+export const getPageSizeForUser = (user : IUser, tableName? : string) => {
```

Comment:
It was made so you can call this function with a parameter that might be undefined which is [how it is used in our code](https://github.com/awslabs/mlspace/blob/main/frontend/src/modules/table/table.tsx#L79). Doing this check once here seemed more reasonable than requiring a null check anywhere this would be called.